pyproject.toml

```diff
@@ -137,3 +135,53 @@ known-first-party = ["tardis", "carsus", "stardis"]
 
 [tool.ruff.lint.pydocstyle]
 convention = "numpy"
+
+[project]
+name = "tardis"
+authors = [{ name = "TARDIS Collaboration", email = "wkerzendorf@gmail.com" }]
+license = { file = "licenses/LICENSE.rst" }
+classifiers = ["License :: BSD-3-Clause"]
+description = "TARDIS - Temperature And Radiative Diffusion In Supernovae"
+readme = { file = "README.rst", content-type = "text/x-rst" }
+requires-python = ">=3.7"
+dependencies = ["astropy"]
+dynamic = ["version"]
+
```

Comment:
I think it usually comes first, can you check well maintained packages, preferably astropy on how they do this

Reply:
yes, I have re arranged the sections